test/testsets.json

```diff
@@ -840,6 +840,13 @@
         "websocket",
         "tls"
       ]
+    },
+    {
+      "name": "test_tls.js",
+      "skip": [
+        "tizenrt"
+      ],
+      "reason": "Error: SSL setup failed (no enough memory for the test)",
```

Comment:
The `test_tls.js` is already exist in the `testsets.json` file. It's enough to modify that entry with `skip` and `reason` information.

Reply:
Note: the comma should be removed at the end of the `reason` line.